browser/sources.gni

```diff
@@ -185,6 +185,9 @@ brave_chrome_browser_deps = [
   "//brave/components/p3a:buildflags",
   "//brave/components/playlist/common/buildflags",
   "//brave/components/privacy_sandbox",
+  "//brave/components/psst/browser/content",
```

Comment:
do we need all three of these added here? Looks like you are only referencing `brave/components/psst/browser/content` and `brave/components/psst/common` directly unless I'm missing something.

Reply:
`registration.cc` needs to register the PSST component.